.github/workflows/build-heretek-rocmfpx.yml

```
@@ -202,5 +244,96 @@ jobs:
+          path: |
+            ${{ env.ARCHIVE_NAME }}
+            build/roadmap-audit.json
+          retention-days: 14
+
+  build-windows:
+    name: Windows MSVC (clang / ${{ matrix.target_family }})
+    # Symmetric to the Linux matrix: build the same engine tree on Windows
+    # so the shipped Matrix tag covers both deployment OS axes. The TheRock
+    # Windows SDK is the same S3 bucket, only the extraction path differs.
+    runs-on: windows-2022
+    if: matrix.target_family == 'apu' || matrix.target_family == 'rdna'
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)